tutorial/README.md

```diff
@@ -0,0 +1,476 @@
+
+![Lesson 3: routing results with alternatives](images/lesson3_resultsWithAlternatives.png)
+
+### Troubleshooting routing
+
+The routing engine returns no result? 
+
+If there is no access, either at origin or destination (no stop nodes near those points), or if there is no service at origin or destination, the error message will be explicit enough. You can change the origin or destination for locations where there is service available.
+
+![Lesson 3: routing results error no access](images/lesson3_resultsErrorNoAccess.png)
+![Lesson 3: routing results error no service](images/lesson3_resultsErrorNoService.png)
+
+If the message says it returned no result, it will hint to look at some parameters values. You can try changing the travel time, access/egress, etc.
+
+![Lesson 3: routing results error no routing](images/lesson3_resultsErrorNoRouting.png)
+
+If you are certain that there should be a route with the given parameters, go back to [lesson 2](#lesson-2-prepare-imported-data-for-calculations) and validate that the scenario contains the expected lines, then refresh calculation data to make sure it is up to date.
+
+## Lesson 4: Visualize accessibility maps
+
+Another type of calculation that can be done with the scenario is the accessibility maps. Accessility maps are isochrones displaying the region accessible in transit within a certain amount of time.
+
+Navigate to the `Accessibility map` panel, by clicking on the `Accessibility map` in the left menu.
+
+![Lesson 4: accessibility map menu](images/lesson4_accessibilityMapMenu.png)
+
+Many of the parameters to set are similar to those of the `Routing`: departure or arrival time and other trip times.
+
+The parameters specific to the accessibility map are the number of polygons and the delta and delta interval. The number of polygons is the number of isochrones that will be drawn. The outermost isochrone is for the maximum travel time specified. And each other isochrone is for a corresponding fraction of the total time. For example, 4 polygons with a 60 minutes total travel time will draw isochrones for 15, 30, 45 and 60 minutes.
+
+As for the delta, it allows to flatten the variabilities at different times by calculating an average of the accessibility a bit before and a bit after the requested time. For example, with the default values of departure time at 8:00 and a delta of 15 minutes and interval of 5 minutes, it will calculate the accessibility at 7:45, 7:50, 7:55, 8:00, 8:05, 8:10 and 8:15. A node `n` may be accessible in 15 minutes at 7:45, 10 minutes at 7:50, 20 minutes à 7:55, etc. The final time for this node will be the average of all the times obtained.
+
+Select the scenario, as for the routing calculation, Then click on the ![calculation button](images/buttonSave.png) calculation button and you should see the result of the route display on the map. Results may take a while to calculate.
```

Comment:
uppercase T in then

Reply:
"Results may take a while to calculate." We can add: "You can select a shorter time to speed up thing if you just want to see a first test"